README.md

```diff
@@ -28,3 +28,38 @@ Open two shell/terminal sessions:
 _Refer to `manage -h` for additional usage information._
 
 Once services are started, visit [http://localhost:8080](http://localhost:8080) to see the app running and visit [http://localhost:8050](http://localhost:8050) to see any outbound mail the app is sending (it won't actually send any email message in the development environment.)
+
+## Deploying to Openshift
+
+**Please make sure you have [Openshift Developer Tools](https://github.com/BCDevOps/openshift-developer-tools/tree/master/bin) installed and available on your path.**
+
+### Param Files
```

Comment:
This looks like it has already been done, so this section is redundant.

Reply:
section removed

Reply:
Looks like the section is still here.

Reply:
weird, sometime git gets out of sync on READMEs for me. Not really sure whats going on but it should be removed now